src/FakeItEasy/IHideObjectMembers.cs

```diff
@@ -6,9 +6,9 @@ namespace FakeItEasy
 
     /// <summary>
     /// Hides standard Object members to make fluent interfaces
-    /// easier to read. Found in the source of Autofac: <see cref="!:https://code.google.com/p/autofac/"/>
+    /// easier to read. Found in the source of Autofac: <see href="https://code.google.com/p/autofac/"/>

```

Comment:
the output XML was only changed in the way you'd expect from this, to include an HREF, which I understand is now allowed